stories/notification-banner/notification-banner.stories.tsx

```diff
@@ -265,9 +265,20 @@ export const ClickableBanner: StoryObj<Component> = {
 
 export const WithMaxCollapsedHeight: StoryObj<Component> = {
     render: () => {
+        const [showParagraph, setShowParagraph] = useState(false);
+
         return (
             <div>
-                <NotificationBanner maxCollapsedHeight={100}>
+                <NotificationBanner maxCollapsedHeight={100}
+                actionButton={{
```

Comment:
can remove the action button story as it's the same as this

the story can be renamed to `HandlingLongContent` too